### PR TITLE
Update ReleaseNotes.md

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Added v2.12.0 template;

### Reason for changes

- Upcoming release;

### Related tickets

- 142565;

#### For the contributors:
Please add your changes (as the commit to the branch) to the list according to the template and previous notes;
Do not add tests-related notes;
Provide the list of the PRs (for all your notes) in the comment for the discussion;

### CODE REVIEW

**Comment by KodiaqQ** [comment]
@alexsu52, @ljaljushkin, @l-bat, @nikita-savelyevv, @andreyanufr, @andrey-churkin, @daniil-lyakhov, @kshpv, @AlexanderDokuchaev fill the document with your changes for the upcoming release, please.

**Review by daniil-lyakhov — APPROVED** [review]
No contributions from my side in this Release

**Review by nikita-savelyevv — APPROVED** [review]

**Comment by kshpv** [comment]
[OpenVINO] Fix ignored_scope for models with IF node. - https://github.com/openvinotoolkit/nncf/pull/2723
Improved ignored_scope logging. - https://github.com/openvinotoolkit/nncf/pull/2723

**Review by kshpv — APPROVED** [review]

**Comment by l-bat** [comment]
(Common): Changed the representation of symmetrically quantized weights from an unsigned integer with a fixed zero-point to a signed data type without zero-point in the weights compression algorithm https://github.com/openvinotoolkit/nncf/pull/2434

**Review by l-bat — APPROVED** [review]

**Comment by KodiaqQ** [comment]
@AlexKoff88, @alexsu52, review please.

**Review by alexsu52 — APPROVED** [review]
LGTM

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
 
+## New in Release 2.12.0
+
+Post-training Quantization:
+
+- Features:
+  - (OpenVINO, PyTorch, ONNX) Excluded comparison operators from the quantization scope for `nncf.ModelType.TRANSFORMER`.
+  - (OpenVINO, PyTorch) Changed the representation of symmetrically quantized weights from an unsigned integer with a fixed zero-point to a signed data type without a zero-point in the `nncf.compress_weights()` method.
+  - (OpenVINO) Extended patterns support of the AWQ algorithm as part of `nncf.compress_weights()`. This allows apply AWQ for the wider scope of the models.
+  - (OpenVINO) Introduced `nncf.CompressWeightsMode.E2M1` as the new precision for the `mode` option of `nncf.compress_weights()`.
```
Comment (AlexKoff88) [inline ReleaseNotes.md:11]:
```suggestion
  - (OpenVINO) Introduced `nncf.CompressWeightsMode.E2M1`  `mode` option of `nncf.compress_weights()` as the new MXFP4 precision (Experimental).
```

**Review by AlexKoff88 — APPROVED** [review]

**Pull request merged**